docs/user_guide/install_server_go.md

```diff
@@ -0,0 +1,135 @@
+# Installing Dragonfly Server
+
+This topic explains how to install the Dragonfly server with **Golang version**.
+
+**NOTE**: The Golang version supernode is **not ready for production usage**. However, you can use it more easily in your test environment.
+
+## Context
+
+Install the SuperNodes in one of the following ways:
+
+- Deploying with Docker: Recommended for quick local deployment and test.
+- Deploying with physical machines: Recommended for production usage.
+
+## Prerequisites
+
+When deploying with Docker, the following conditions must be met.
+
+Required Software | Version Limit
+---|---
+Git|1.9.1+
+Docker|1.12.0+
+
+When deploying with physical machines, the following conditions must be met.
+
+Required Software | Version Limit
+---|---
+Git|1.9.1+
+Golang|1.10.2+
+Nginx|0.8+
+
+## Procedure - When Deploying with Docker
+
+### Get Supernode image
+
+You can build your own supernode image.
+
+1. Obtain the source code of Dragonfly.
+
+    ```sh
+    git clone https://github.com/dragonflyoss/Dragonfly.git
+    ```
+
+2. Enter the project directory.
+
+    ```sh
+    cd Dragonfly
+    ```
+
+3. Build the Docker image.
+
+    ```sh
+    TAG="test"
+    make docker-build-supernode DF_VERSION=$TAG
+    ```
+
+4. Obtain the latest Docker image ID of the SuperNode.
+
+    ```sh
+    docker image ls|grep 'supernode' |awk '{print $3}' | head -n1
+    ```
+
+Or you can get it from [DockerHub](https://hub.docker.com/) directly.
+
+1. Obtain the latest Docker image ID of the SuperNode.
+
+    ```sh
+    docker pull dragonflyoss/supernode:0.4.0
+    ```
```

Comment:
move this way to the top of this section

Reply:
Done.